.eslintrc.js

```diff
@@ -39,6 +39,13 @@ module.exports = {
         '@typescript-eslint/no-use-before-define': 'warn',
         'import/extensions': 'off',
         'consistent-return': 'off',
+        'react/function-component-definition': [
```

Comment:
Can this be deleted so it uses it as-is from the base config, or is there something here making it more strict?

Reply:
It does not use arrow functions settings by default:

> This rule is aimed to enforce consistent function types for function components. By default it prefers function declarations for named components and function expressions for unnamed components.